core/src/main/java/com/coderanch/blackjack/Cards.java

```diff
@@ -20,6 +21,8 @@
  */
 final class Cards {
 
+    private static final Random RANDOM = new Random();
```

Comment:
Don't declare non-constants as static fields. `Random` is a service, so inject it as a constructor parameter into normal classes, or in the case of static utility classes (such as this one) inject it directly into the static method:

```java
static Stack<Card> getShuffledStandardDeck(Random generator) {
```
The reason for this is that it makes unit-testing much easier, because you can inject `Random` implementations that return a known sequence of 'random' values.